lms/envs/minimal.yml

```diff
@@ -20,6 +20,12 @@ MKTG_URL_LINK_MAP: {}
 MKTG_URL_OVERRIDES: {}
 REST_FRAMEWORK: {}
 
+# We need this because otherwise ./lms/djangoapps/certificates/migrations/0003_data__default_modes.py will fail.
+#
```

Comment:
Nit: I personally don’t like empty lines between a comment and what is being commented on, because I’m always a little unsure of whether it really applies to the code below, or was a leftover comment that someone failed to delete or move. Using the actual name instead of “this”, or getting rid of the empty line, both resolve this for me. For all I know, I’m the only one with this issue, so feel free to resolve with or without changing.